File: algo/3.py
# Luana Amorim Lima
# Rafael Nascimento 
# Joao Victor Pereira Cavalcante 

n = 100
arr = input().split()


def g(j, i):
	if j < i:
		return

	g(j-1, i)
	if int(arr[i]) < int(arr[j]):
		tmp = arr[i]
		arr[i] = arr[j]
		arr[j] = tmp


def f(i):
	if i < 0:
		return 

	f(i-1)
	g(n-1, i)

f(n-1)

print(arr)

File: algo/3.c
//Luana Amorim Lima
#include <stdio.h>
#include <string.h>

int m[3][3], n;

int main(void)
{
	for (int i = 0; i < 3; i++)
		for (int j = 0; j < 3; j++)
			scanf("%d", &m[i][j]);

	scanf("%d", &n);

	for (int i = 0; i < 3; i++)
	{
		for (int j = 0; j < 3; j++)
		{
			m[i][j] *= n;
			printf("%d ", m[i][j]);
		}
		printf("\n");
	}
}